src/main/resources/edu/cornell/kfs/pdp/businessobject/datadictionary/PayeeACHAccount.xml

```diff
@@ -89,254 +38,93 @@
   </bean>
-
-<!-- Business Object Lookup Definition -->
-
-
-  <bean id="PayeeACHAccount-lookupDefinition" parent="PayeeACHAccount-lookupDefinition-parentBean"/>
-
-  <bean id="PayeeACHAccount-lookupDefinition-parentBean" abstract="true" parent="LookupDefinition">
-    <property name="title" value="Payee ACH Account Lookup"/>
-    <property name="lookupableID" value="payeeACHAccountLookupable" />    
-    
-    <property name="defaultSort">
-      <bean parent="SortDefinition">
-        <property name="attributeNames">
-          <list>
-            <value>achAccountGeneratedIdentifier</value>
-          </list>
-        </property>
-      </bean>
-    </property>
-    <property name="lookupFields">
-      <list>
-        <bean parent="FieldDefinition" p:attributeName="payeeIdentifierTypeCode"/>
-        <bean parent="FieldDefinition" p:attributeName="payeeIdNumber"/>              
-        <bean parent="FieldDefinition" p:attributeName="bankRoutingNumber"/>
-        <bean parent="FieldDefinition" p:attributeName="bankAccountNumber"/>
-        <bean parent="FieldDefinition" p:attributeName="bankAccountTypeCode"/>
-        <bean parent="FieldDefinition" p:attributeName="payeeEmailAddress"/>
-        <bean parent="FieldDefinition" p:attributeName="payeeName"/>
-        <bean parent="FieldDefinition" p:attributeName="payeePrincipalName"/><!-- CU Customization: Added principal name. -->
-        <bean parent="FieldDefinition" p:attributeName="achTransactionType"/>
-        <bean parent="FieldDefinition" p:attributeName="active"/>
-      </list>
-    </property>
-    <property name="resultFields">
+  
+  <bean parent="DataDictionaryBeanOverride">
+    <property name="beanName" value="PayeeACHAccount-lookupDefinition" />
+    <property name="fieldOverrides">
       <list>
-        <bean parent="FieldDefinition" p:attributeName="payeeName"/>
-        <bean parent="FieldDefinition" p:attributeName="payeeEmailAddress"/>
-        <bean parent="FieldDefinition" p:attributeName="payeeIdentifierTypeCode"/>
-        <bean parent="FieldDefinition" p:attributeName="payeePrincipalName"/><!-- CU Customization: Added principal name. -->
-        <bean parent="FieldDefinition" p:attributeName="achTransactionType"/>
-        <bean parent="FieldDefinition" p:attributeName="active"/>
+        <bean parent="FieldOverrideForListElementInsert">
+          <property name="propertyName" value="lookupFields" />
+          <property name="propertyNameForElementCompare" value="attributeName" />
+          <property name="element">
+            <bean parent="FieldDefinition" p:attributeName="bankAccountTypeCode" />
+          </property>
+          <property name="insertAfter">
+            <list>
+              <bean parent="FieldDefinition" p:attributeName="payeeEmailAddress" />
+              <bean parent="FieldDefinition" p:attributeName="payeeName" />
+              <bean parent="FieldDefinition" p:attributeName="payeePrincipalName" />

```

Comment:
I'm a little confused, based on the diff, payeePrincipalName is our only custom field.  Are payeeEmailAddress and payeeName already part of bse code's lookup fields?

Reply:
Good question. payeePrincipalName is our custom field but we also customized the lookup to add other existing fields to the search fields.

Reply:
Basically I tried keeping the lookup and inquiry looking the same after cleaning up this file to use overrides.

Reply:
ah, thank you, this should be fine